translation.json

```diff
@@ -0,0 +1,20 @@
+{
+  "widgets": {
+    "notificationInbox": {
+      "actions": {
+        "loadMore": "Load more"
+      },
+      "noNotifications": {
+        "title": "No notifications yet",
+        "instruction": "Check again later for updates"
+      },
+      "errors": {
+        "loadingFailed": "There was an error loading notifications"
+      },
+      "toolbar": {
+        "unread": "unread",
```

Comment:
The `<span class="unread-count">\{{count}}</span>` container should also be included because "x unread" might be translated with the number at the end for some languages.